streams/src/test/java/org/apache/kafka/test/InternalMockProcessorContext.java

```diff
@@ -306,10 +304,6 @@ public void restore(final String storeName, final Iterable<KeyValue<byte[], byte
         restoreListener.onRestoreEnd(null, storeName, 0L);
     }
 
-    public void close() {
-        metrics.close();
```

Comment:
`Metrics.close()` doesn't do anything we care about in tests, so this just adds unnecessary complexity and an unnecessary dependency on `StreamsMetricsImpl#registry()`. I elected just to remove it; WDYT?

Reply:
seems reasonable, but since `metrics` is an instance field on `AbstractProcessorContext` do we at least want to provide access to the `metrics` fields for any future tests?